routes/AddLogViewMapEntry.js

```diff
@@ -0,0 +1,38 @@
+var express = require("express");
```

Comment:
Since we will have more than one logging route, can we start a subfolder for this and move it to:
routes/Logging/AddLogViewMapEntry.js?